### ISSUE TITLE
feat: AutoSync connected contracts when syncState

### ISSUE THREAD

**Issue by twilson63** [warp-contracts/warp#333]
DRE nodes retain latest state of contracts that use readState and internalWrites so that browser-based clients do not have to evaluate every interaction just to do a writeInteraction.

This feature request is to autoSync all connected contracts when a `syncState` is called.

```ts
const warp = WarpFactory.forMainnet();

const contract = await warp
    .contract('pbabEjmdaqOvF-yTkFhs5i2lbmmbC6s4NrUqM_8eAYE')
    .syncState('https://dre-1.warp.cc/contract', {validity: true}); // auto sync all states connected to contract

const {sortKey} = await warp.stateEvaluator.latestAvailableState('pbabEjmdaqOvF-yTkFhs5i2lbmmbC6s4NrUqM_8eAYE');

console.log(sortKey);
```

**Comment by ppedziwiatr** [comment]
@twilson63 , thanks for the report - we will try to introduce this feature next week!

**Comment by ppedziwiatr** [comment]
released in 1.2.55
